### PR TITLE
feat: Event listing for administration

### PR DESCRIPTION
**Pull request by ipeglin**
## Changes
A list view of events stored in the database. The list shows a simple-detailed list of all event properties, and provides the option to edit or delete each event.

### CODE REVIEW

**Review by AasmundN — CHANGES_REQUESTED** [review]
Looks super smooth! Can we set the overflow of the two boxes mentioned below to scroll? Think it looks ugly when the elements are all bunched up, as they will be on mobile.

File: pages/admin/events.vue [hunk 1]
```diff
@@ -1,3 +1,403 @@
...
+            value="ascending"
+          >
+            {{ $t('admin.events.filterorder.ascending') }}
+          </VChip>
+        </VChipGroup>
+      </div>
+
+      <VSpacer />
+
+      <div>
```
Comment (AasmundN) [inline pages/admin/events.vue:184]: This div

File: pages/admin/events.vue [hunk 2]
```diff
@@ -1,3 +1,403 @@
...
+          @click="navigateTo('/event/edit')"
+        >
+        </VBtn>
+      </div>
+    </div>
+
+    <VDivider class="mb-5 mt-2" />
+
+    <!-- Pagination -->
+    <div class="d-flex align-center justify-space-between flex-wrap">
```
Comment (AasmundN) [inline pages/admin/events.vue:265]: And this one

**Comment by ipeglin** [comment]
> Looks super smooth! Can we set the overflow of the two boxes mentioned below? Think it looks ugly when the elements are all bunched up, as they will be on mobile.

Not sure if I fully understand. Would you like the overflow to be set to hidden?

**Comment by AasmundN** [comment]
I meant the wrapping in these two elements. But it's not imortant.

overflow

**Review by AasmundN — APPROVED** [review]

**Pull request merged**